pbf_file_size_estimation/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = '0.0.7'
+__version__ = '1.0.0'
```

Comment:
Just because it is used somewhere, doesn't mean it should be considered as stable :smirk: 

Semantic versioning makes sense, though.